src/main/app/claims/models/claim-states/states-paid-states.ts

```diff
@@ -0,0 +1,3 @@
+export enum StatesPaidStates {
+  STATES_PAID = 'states-paid'
```

Comment:
States paid can be via full defence or part admit - is a single state enough?

Reply:
Full defence has states called FD_ALREADY PAID, FD_ALREADY_PAID_RESPONSE, , FD_ALREADY_PAID_ACCEPT, and FD_ALREADY_PAID_REJECT.

Reply:
So is this just for part admit states paid? What about the substates under that?

Reply:
Correct. I will be removing this states paid state from this story as it (and its substates) are being addressed in a separate story (https://tools.hmcts.net/jira/browse/ROC-5267, I'll attach a screenshot below of my WIP for that story), this state was already there when I started work on this story :)

<img width="580" alt="Screen Shot 2019-06-27 at 14 17 18" src="https://user-images.githubusercontent.com/32717248/60269413-52c43080-98e6-11e9-81e3-9cf70a3c7d7c.png">